tequilapi/contract/pageable.go

```diff
@@ -55,20 +54,9 @@ func (q *PaginationQuery) Bind(request *http.Request) *validation.FieldErrorMap
 	errs := validation.NewErrorMap()
 
 	qs := request.URL.Query()
-	if qStr := qs.Get("page_size"); qStr != "" {
-		if qVal, err := parseInt(qStr); err != nil {
-			errs.ForField("page_size").Add(err)
-		} else {
-			q.PageSize = *qVal
-		}
-	}
-	if qStr := qs.Get("page"); qStr != "" {
-		if qVal, err := parseInt(qStr); err != nil {
-			errs.ForField("page").Add(err)
-		} else {
-			q.Page = *qVal
-		}
-	}
+
+	NewQueryParse(qs, "page_size").BindInt(&q.PageSize, errs)
```

Comment:
I would simple function call instead struct, `BindInt` is not reusable anyways, as it's conditions is suitable just for: query params, just for optional ints